Xamarin.PropertyEditing.Windows/NumericUpDownControl.cs

```diff
@@ -62,6 +62,11 @@ internal abstract class NumericUpDownControl<T>
 		: Control
 		where T : struct, IComparable<T>
 	{
+		static NumericUpDownControl ()
+		{
+			FocusableProperty.OverrideMetadata (typeof (NumericUpDownControl<T>), new FrameworkPropertyMetadata (false));
```

Comment:
Don't we want the Focusable to be *true* in this case, while PropertyEditorControl is *false*?